hypergan/graph/graph.py

```diff
@@ -77,6 +80,18 @@ def create_z_encoding(self):
         z_encoded = tf.concat(axis=1, values=encoders)
         self.gan.graph.z_encoded = z_encoded
 
+        encoders = []
+        with(tf.variable_scope("encoder2", reuse=False)):
```

Comment:
Maybe we should make a way to create new encodings.  I could see wanting to sample that probability distribution repeatedly.